### ISSUE TITLE
Get into building a translation in Korean.

### ISSUE THREAD

**Issue by Undecember** [DreamExposure/DisCal-Discord-Bot#75]
**What language are you requesting translations for?**
Korean
**Do you know anyone that can help translate to this language?**
I will.
**Additional context**
Is it adaptable to make Korean translation? I'm worried what if this discord bot isn't compatible to Korean. If not, I want to translate it.

**Comment by NovaFox161** [comment]
Yes, you are more than welcome to translate the bot to Korean, instructions for the current translation process is on the README file.

**Comment by NovaFox161** [comment]
@Undecember ny updates on this? I know I plan on redoing the way translations are integrated into the bot (so that its easier to do them) but currently haven't started work on that.

**Comment by Undecember** [comment]
Oh I had hard days so that I couldn't work on it. um.. actually forgot to do. I'll finish it until this weekend.

**Comment by NovaFox161** [comment]
Not a problem at all <3 was just wondering where it was since I'm trying to get github issues sorted out.
